isonet/protocols/protocol_tomo_reconstruction.py

```diff
@@ -272,7 +272,7 @@ def prepareProjectStep(self):
             tomoName = tomo.getTsId()
             tomoLnName = os.path.join(self.tomoPath, tomoName + '.mrc')
             if not os.path.exists(tomoLnName):
-                os.link(tomofn, tomoLnName)
+                os.symlink(tomofn, tomoLnName)
```

Comment:
I recommend to replace symlink by pyworkflow > utils > path.py method called createLink, as it is more robust

Reply:
This is already resolved and merged in a separate PR